src/components/UserConsent/index.tsx

```diff
@@ -24,7 +19,7 @@ type ActionsProps = {
 }
 
 const Actions: FunctionComponent<ActionsProps> = ({ onAccept, onDecline }) => {
-  const { translate } = useContext(LocaleContext)
+  const { translate } = useLocales()
```

Comment:
Is the rule of thumb going forward to use `userLocales()` instead of `useContext(LocaleContext)` for a `FunctionComponent`?

Reply:
Yes, there're 2 reasons:

1. In the future we don't want to expose `LocaleContext` publicly to the consumers, just `useLocales` is enough.
2. `useContext(LocaleContext)` returns `WithLocalisedProps | undefined`, while `useLocales()` returns `WithLocalisedProps` or throws an error. This mechanism is to assure that every consumer is properly wrapped inside `<LocaleProvider />`.